src/ext/java/org/opentripplanner/ext/legacygraphqlapi/generated/LegacyGraphQLDataFetchers.java

```diff
@@ -587,7 +586,7 @@ public interface LegacyGraphQLQueryType {
 
     public DataFetcher<Connection<NearbyStop>> stopsByRadius();
 
-    public DataFetcher<Iterable<FareRuleSet>> ticketTypes();
+    public DataFetcher<Iterable<org.opentripplanner.ext.legacygraphqlapi.model.LegacyGraphQLTicketType>> ticketTypes();
```

Comment:
Do not update this file manually, but instead edit `src/ext/java/org/opentripplanner/ext/legacygraphqlapi/generated/graphql-codegen.yml` and re-generate this file

Reply:
There is more info on how to do it here: https://github.com/opentripplanner/OpenTripPlanner/blob/ef8d73a166b0e408f826d0dcfa2bb167bc7e986d/src/ext/java/org/opentripplanner/ext/legacygraphqlapi/generated/README.md

I think the docs are also a little bit outdated. Perhaps @optionsome can help you update it.

Reply:
The `TODO fix this ` and below could be removed, as now the original instructions work again